packages-exp/auth-exp/src/platform_browser/strategies/redirect.ts

```diff
@@ -34,91 +34,163 @@ import { User, UserCredential } from '../../model/user';
 import { _withDefaultResolver } from '../popup_redirect';
 import { AbstractPopupRedirectOperation } from './abstract_popup_redirect_operation';
 
+/**
```

Comment:
For both redirect/popup, the `@param` declaration for `resolver` should say it's optional if the resolver was provided in the `{@link initializeAuth()}` call, _or_ `getAuth()` was used

Reply:
done